run.exs

```diff
@@ -17,19 +17,28 @@ alias Xander.Query
 # Default config connects via a local UNIX socket
 config = Config.default_config!(socket_path)
 
+queries = [
+  :get_epoch_number,
+  :get_current_era,
+  :get_current_block_height
+]
+
 case Query.start_link(config) do
   {:ok, pid} ->
     IO.puts("Successfully connected to Cardano node")
 
-    case Query.run(pid, :get_current_era) do
-      {:ok, current_era} ->
-        IO.puts("Current Cardano era: #{inspect(current_era)}")
+    for query <- queries do
+      case Query.run(pid, query) do
```

Comment:
this script is a more thorough version of the ones that were deleted in this PR. Is that right?

Reply:
Yep. The mix tasks were using the `Application` that's no longer around since we're not doing Supervisors.